engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/bulk/content-hosts-bulk-system-purpose-modal.controller.js

```diff
@@ -0,0 +1,93 @@
+/**
+ * @ngdoc object
+ * @name  Bastion.content-hosts.controller:ContentHostsBulkSystemPurposeModalController
+ *
+ * @requires $scope
+ * @requires $uibModalInstance
+ * @requires HostBulkAction
+ * @requires Organization
+ * @requires CurrentOrganization
+ * @requires Notification
+ * @requires hostIds
+ *
+ * @description
+ *   A controller for providing bulk action functionality for setting system purpose values
+ */
+angular.module('Bastion.content-hosts').controller('ContentHostsBulkSystemPurposeModalController',
+    ['$scope', '$uibModalInstance', 'HostBulkAction', 'Organization', 'CurrentOrganization', 'Notification', 'hostIds',
+    function ($scope, $uibModalInstance, HostBulkAction, Organization, CurrentOrganization, Notification, hostIds) {
+
+        $scope.purposeAddonsList = function () {
+            var addonsList = [];
+            var purposeAddonsCount;
```

Comment:
This isn't used outside or inside this function, so it's not needed anymore
```suggestion
```

Reply:
Removed